Jobs/PrGate.yml

```diff
@@ -35,6 +35,9 @@ parameters:
   displayName: Perform Stuart PR Evaluation
   type: boolean
   default: true
+- name: calculate_code_coverage

```

Comment:
Can you add the `type` to keep consistent with the existing parameter format?

Reply:
added a type to calculate_code_coverage as requested! Thanks